dev/io.openliberty.io.smallrye.reactive.messaging-provider4/src/io/smallrye/reactive/messaging/providers/extension/EmitterImpl.java

```diff
@@ -0,0 +1,48 @@
+package io.smallrye.reactive.messaging.providers.extension;
```

Comment:
Add a copyright header and statements about where the original file came from and why we changed it